managers/engineering_manager.md

```diff
@@ -1,17 +1,19 @@
 Engineering Manager
 ===================
 
-## The Engineering Manager is accountable for building high performing teams.
+## The Engineering Manager is accountable for building high performing teams
 
 ### Accountabilities
 
-* *If the team has high cycle time, it’s on the EM*
-* *If the team isn’t continuously improving their process to deliver more effectively & reduce bottlenecks, it’s on the EM*
-* *If the team is not communicating well or working well together it’s on the EM.*
-* *If the engineers who report to them don’t have the right skillset to tackle their problems it’s on the EM*
-* *If the engineers who report to them aren’t focussed on their work it’s on the EM*
-* *If engineers that report to them have performance problems that impact teams being able to deliver, it’s on the EM*
-* *If engineers that report to them don’t have personal goals & career development plans that line up with team goals, it’s on the EM*
+It's the responsibility of the EM to ensure all of the following for their teams and direct reports:
```

Comment:
Similar feedback here @adsteel - I loathed the `it's on the EM` at the end of every line - line 8 conveys the sentiment but conscientiously